flink-contrib/flink-storm-examples/pom.xml

```diff
@@ -364,6 +363,18 @@ under the License.
 							</transformers>
 						</configuration>
 					</execution>
+					<execution>
+						<id>shade-flink</id>
```

Comment:
The maven's philosophy is 'one artifact per project'. These projects don't follow this way and produces several jar's artifacts :)
Shade plugin should produce flink-storm-streaming_2.10-1.2-SNAPSHOT-shaded.jar but it doesn't do it. It is a problem for maven install plugin that wants to have such jar by default. And it leads to failure on install phase without such configuration changes. For this reason I added such changes.

So I propose don't collect test coverage for these projects (flink-storm and streaming examples)  at all. What do you think?